models/nft/whale.sql

```diff
@@ -0,0 +1,39 @@
+with nft_trades as (
+  select *
+  from {{ ref('nft_trades') }}
+),
+
+contracts as (
+    select distinct address
+    from {{ ref('stg_contracts') }}
+),
+
+stat as (
+    select *
+    from (
+        select 
+            buyer as address, 
+            sum(usd_amount) as amount
+        from (
+            select distinct buyer, usd_amount, nft_token_id, nft_contract_address
+            from (
+                select buyer, nft_contract_address, nft_token_id, usd_amount, 
+                    row_number() over(partition by nft_contract_address, nft_token_id order by block_time desc) as rank
+                from nft_trades
+            )
+            where rank=1
+        )
+        group by 1
+    )
+    where amount>1000000
+    order by amount desc
+)
+
+
+select 
+    stat.address, 
+    'Whale' as label,
```

Comment:
Suggest to give a specific name to this whale label, like 'NFT whale' or 'NFT Millionaire' since we will have other whale labels in the future.